lib/pages/app_grid/app_grid_dashboard_view_style.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:fluffychat/domain/model/app_grid/linagora_applications.dart';
+
+class AppGridDashboardViewStyle {
+  static const double sizIcAppGrid = 48.0;
+
+  static double widthAppGrid(LinagoraApplications linagoraApplications) {
```

Comment:
Found this also appear inside app_grid_dashboard_overlay_style`. How about move to 1 find and reuse? WDYT?